src/main/java/org/cactoos/collection/Skipped.java

```diff
@@ -52,16 +51,7 @@ public Skipped(final int skip, final T... src) {
      * @param src Source iterable
      */
     public Skipped(final int skip, final Iterable<T> src) {
-        this(skip, new CollectionOf<>(src));
-    }
-
-    /**
-     * Ctor.
-     * @param skip How many to skip
-     * @param src Source collection
-     */
-    public Skipped(final int skip, final Collection<T> src) {
-        super(() -> new CollectionOf<>(
+        super(new Sticky<>(
```

Comment:
@victornoel please, use the same indentation here which you used bellow (IMHO it's better).